README.rst

```diff
@@ -96,6 +96,12 @@ For Linux or OSX run::
 
     $ eval $(aws-okta-processor authenticate --environment --user <user_name> --organization <organization>.okta.com)
 
+On Unix systems is allowed to pass a `--target-shell` in order to change the
```

Comment:
Lets change `On Unix systems is allowed to pass...` to `On Unix systems pass...`

Reply:
Thank you. I will update that!

Reply:
Done :heavy_check_mark: